Sdk/Conversation/SwrveConversationStyler.m

```diff
@@ -79,8 +90,9 @@ +(void) styleButton:(SwrveConversationUIButton*)button withStyle:(NSDictionary*)
     }
     NSString *bgHexColor = [self colorFromStyle:[style objectForKey:kSwrveKeyBg] withDefault:kSwrveDefaultColorBg];
     UIColor *bgUIColor = [self convertToUIColor:bgHexColor];
+    float borderRadius = [self convertBorderRadius:[[style objectForKey:kSwrveKeyBorderRadius] floatValue]];
```

Comment:
Border could not be there, we have to accept nil and return a 0 border radius (our current default)